src/checkout-buttons/strategies/paypal-button-strategy.ts

```diff
@@ -87,8 +88,12 @@ export default class PaypalButtonStrategy extends CheckoutButtonStrategy {
             throw new NotInitializedError(NotInitializedErrorType.CheckoutButtonNotInitialized);
         }
 
-        return actions.request.post('/api/storefront/paypal-payment/', { merchantId })
-            .then(res => res.id)
+        return actions.request.post(`${this._host || ''}/api/storefront/paypal-payment/`, {
```

Comment:
Do you still need `|| ''` if `host` has a default value of `''`?

Reply:
Nope, good catch.